CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 # Changelog
 
-## 24.5.3 [#734](https://github.com/openfisca/openfisca-core/pull/734)
+### 24.5.4 [#737](https://github.com/openfisca/openfisca-core/pull/737)
+
+- Fix a bug on encoding for OpenFisca variables of string type
+- Details:
+  * Set variable value encoding to `numpy.unicode_` in python 2.7 and python 3.7
```

Comment:
I think an explanation of the situation would be very useful for other contributors.